specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2021-05-13-preview/desktopvirtualization.json

```diff
@@ -3601,6 +3609,283 @@
           },
           "type": "string",
           "description": "Enabled to allow this resource to be access from the public network"
+        },
+        "sessionHostConfiguration": {
+          "$ref": "#/definitions/SessionHostConfigurationProperties",
+          "description": "The session host configurations of HostPool."
+        },
+        "sessionHostComponentUpdateConfiguration": {
+          "$ref": "#/definitions/SessionHostComponentUpdateConfigurationProperties",
+          "description": "The session host configuration for updating agent, monitoring agent, and stack component."
+        }
+      }
+    },
+    "SessionHostConfigurationProperties": {
+      "description": "Session host configurations of HostPool.",
+      "type": "object",
+      "properties": {
+        "vMSizeId": {
```

Comment:
vmSizeId

Reply:
in all of the swaggers its defined vMSizeId